### PR TITLE
makes toolspeeds 0 all across the board except for welders

### PR DESCRIPTION
**Pull request by ghost**
[Changelogs]: makes toolspeeds 0 all across the board except for welders

:cl: zamolxius
tweak: tweaked toolspeeds
balance: rebalanced toolspeeds
fix: fixed /tg/'s shitty balancing
/:cl:

[why]: # pre rebase feature

### CODE REVIEW

**Review by Carbonhell — DISMISSED** [review]

**Comment by Carbonhell** [comment]
+1

**Comment by GuyonBroadway** [comment]
You forgot the traitor tools and the CE's tools

**Comment by ghost** [comment]
yeah ill do them and multitools

**Comment by Dinopeaks** [comment]
-1 there should be some speed to them, also would make CE tools a bit more useless

**Comment by ghost** [comment]
CE tools let you not need to juggle the tools in your belt/bag, you have wrench, wirecutter, crowbar, screwdriver all in only 2 slots/hands. still a big improvement.

**Comment by ghost** [comment]
also welders are unaffected

**Comment by Dinopeaks** [comment]
yea the need for juggling is gone but the CE tools are always suppose to be faster

**Comment by ghost** [comment]
you save like 0.5s a lot of time normally spent moving the mouse to the bag, clicking the item, then clicking the object, then pressing E

**Comment by ghost** [comment]
plus they look cool

**Comment by Carbonhell** [comment]
+1 again

**Comment by Dinopeaks** [comment]
>looking cool, they should be faster just not instant

File: hippiestation/code/game/objects/items/tools.dm [hunk 1]
```diff
@@ -10,4 +10,85 @@
...
+/obj/item/wrench/cyborg
+	toolspeed = 0
+
+/obj/item/wrench/abductor
+	toolspeed = 0
+
+/obj/item/wrench/power
+	toolspeed = 0
+
+/obj/item/wrench/syndicate
```
Comment (GuyonBroadway) [inline hippiestation/code/game/objects/items/tools.dm]: /obj/item/wrench/syndicate is a hippie based item, you can edit its file directly, no need to modularize this line.

File: hippiestation/code/game/objects/items/tools.dm [hunk 2]
```diff
@@ -10,4 +10,85 @@
...
+/obj/item/screwdriver/abductor
+	toolspeed = 0
+
+/obj/item/screwdriver/power
+	toolspeed = 0
+
+/obj/item/screwdriver/cyborg
+	toolspeed = 0
+
+/obj/item/screwdriver/nuke
```
Comment (GuyonBroadway) [inline hippiestation/code/game/objects/items/tools.dm]: /obj/item/screwdriver/nuke has already been mirrored elsewhere, you can edit the mirrored file directly, no need to modularize this line.

**Review by GuyonBroadway — CHANGES_REQUESTED** [review]

File: hippiestation/code/game/objects/items/tools.dm [hunk 3]
```diff
@@ -10,4 +10,85 @@
...
+/obj/item/wirecutters/abductor
+	toolspeed = 0
+
+/obj/item/wirecutters/cyborg
+	toolspeed = 0
+
+/obj/item/wirecutters/power
+	toolspeed = 0
+
+/obj/item/wirecutters/syndicate
```
Comment (GuyonBroadway) [inline hippiestation/code/game/objects/items/tools.dm]: /obj/item/wirecutters/syndicate is a hippie based item, you can edit its file directly, no need to modularize this line.

**Review by GuyonBroadway — DISMISSED** [review]

**Comment by GuyonBroadway** [comment]
nevermind about the nukie scredriver, I was mistaken.

**Comment by ghost** [comment]
ready for merge

**Comment by dreamcrab** [comment]
+9

**Comment by Jujumatic** [comment]
@Dinopeaks you have no power here, mr HEAD MENTOR !!!!!!!!!

**Review by Jujumatic — APPROVED** [review]
fuck it why not lol

**Comment by Dinopeaks** [comment]
Just saying it seems broken

**Pull request merged**